### PR TITLE
N29: Fish Fight

### PR DESCRIPTION
**Pull request by erlend-sh**
_Part of #888_

### CODE REVIEW

File: content/news/029/index.md [hunk 1]
```diff
@@ -67,6 +67,27 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+![Fish demo scene](fishfight-poster.png)
+
+Fish Fight ([GitHub][fish-github], [Discord][fish-discord],
+[website][fish-website]) 
+
+Fish Fight is a tactical 2D shooter, played by up to 4 players.
+It is also a 2D-pixels-platformer game engine optimized for modding.
+
+In-game level editor was [released][fish-editor], along with an [editor tutorial][fish-tutorial].
+A retrospective devlog was posted: [Fish Fight's past, present and future][fish-announce]
```
Comment (AngelOnFira) [inline content/news/029/index.md:81]:
```suggestion
A retrospective devlog was posted: [Fish Fight's past, present, and future][fish-announce].
```

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
Image is a bit too large, converting to jpg should do the trick 👍

**Comment by erlend-sh** [comment]
Converted to JPG. I dunno how to get this PR resolved now though. Maybe I should just open a new one..

**Comment by AngelOnFira** [comment]
I you create another commit where you delete the image, it will be fine since the commits get squashed :) Although the image is still a bit large. I can make a commit that cleans this up :)

**Review by AngelOnFira — APPROVED** [review]

File: content/news/029/index.md [hunk 2]
```diff
@@ -213,6 +213,27 @@ read a twitter thread that brifly covers how it
 [country-slice-twitter]: https://twitter.com/anastasiaopara/
 [country-slice-twitter-opt]: https://twitter.com/anastasiaopara/status/1472627194409230343
 
+### Fish Fight
+
+![Fish demo scene](fishfight-poster.png)
+
+Fish Fight ([GitHub][fish-github], [Discord][fish-discord],
+[website][fish-website])
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:221]:
```suggestion
[website][fish-website])
```

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by erlend-sh** [comment]
Thanks for the assist! 🙏 ❤️

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**